samples/04_gis_analysts_data_scientists/translating_story_map_from_one_language_to_another.ipynb

```diff
@@ -0,0 +1,4030 @@
+{
```

Comment:
<p>why do we need two different gis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>We will be picking up a publicly available ArcGIS item and clone it to <code>home GIS</code> and then translating the contents of it. The first gis will be used to pick/get the item and the second gis will be used to clone the item and translate it.</p>